bridge/engine/EngineThreadContext.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include "EngineFunction.h"
+#include "concurrency/MpmcQueue.h"
+
+namespace bridge
+{
+
+class EngineThreadContext
```

Comment:
Easy way to WT to "post" task to EngineThread without create a round trip of specific message to MixerManager and MixerManager to Engine

Reply:
Here I think we should call it what it is, a sort of TaskQueue or something.

Reply:
Maybe not in this same PR, but we should consider using this in MixerManager and Engine if we think this is the way to handle pending tasks / jobs. 
We have 3 methods now:
- POD type union messages
- Job based class per method call
- Task / serialized function call queue
We should perhaps replace the POD type one. This could have been solved by posting a POD type onto the engine command queue afaik, but it would have disadvantages like limitations on the objects passed in arguments.

Reply:
What is actually the problem of making a generic Job that can take on that "on stack" Function type?
I assume there are some catches in the details.

Reply:
Then we would have one JobQueue to pull from instead of two different ones.